upgrading/downgrade.adoc

```diff
@@ -63,24 +58,23 @@ On node and master hosts:
 /etc/origin/node/node-config.yaml
 ----
 +
-On etcd hosts (including masters that have etcd co-located on them):
+On etcd hosts, including masters that have etcd co-located on them,:
 +
 ----
 /etc/etcd/etcd.conf
-/backup/etcd-xxxxxx/backup.db
 ----
 
 [[downgrade-shutting-down-the-cluster]]
-== Shutting Down the Cluster
+== Shutting down the cluster
 
-. On all master and node hosts:
+. On all master and node hosts, run:
 +
 ----
 # systemctl stop atomic-openshift-node
 ----
 
 [[downgrade-removing-rpms]]
-== Removing RPMs and Static Pods
+== Removing RPMs and static pods
 
 The **-excluder* packages add entries to the exclude directive in the host’s
 *_/etc/yum.conf_* file when installed.
```

Comment:
This sentence seem not related with following context. Please confirm.

Reply:
You're right. Thank you!